tools/gfx/persistent-shader-cache.cpp

```diff
@@ -39,25 +40,26 @@ PersistentShaderCache::PersistentShaderCache(const Desc& inDesc)
 void PersistentShaderCache::loadCacheFromFile()
 {
     ComPtr<ISlangBlob> indexBlob;
-    if (SLANG_FAILED(desc.shaderCacheFileSystem->loadFile(desc.cacheFilename.getBuffer(), indexBlob.writeRef())))
+    if (SLANG_FAILED(desc.shaderCacheFileSystem->loadFile(desc.cacheFilename, indexBlob.writeRef())))
     {
         // Cache index not found, so we'll create and save a new one.
         if (mutableShaderCacheFileSystem)
         {
-            mutableShaderCacheFileSystem->saveFile(desc.cacheFilename.getBuffer(), nullptr, 0);
+            mutableShaderCacheFileSystem->saveFile(desc.cacheFilename, nullptr, 0);
             return;
         }
         // Cache index not found and we can't save a new one due to the file system being immutable.
         return;
     }
 
-    String indexString;
-    File::readAllText(desc.cacheFilename, indexString);
+    String indexString = String((char*)indexBlob->getBufferPointer());
 
     List<UnownedStringSlice> lines;
     StringUtil::calcLines(indexString.getUnownedSlice(), lines);
     for (auto line : lines)
     {
+        if (line == "")
```

Comment:
I don't think we need this. Just `continue` if size of `digests` after `split` is not 2.

Reply:
Changed locally, will be in next push